command/git/commands.go

```diff
@@ -31,7 +31,7 @@ func (g *Git) RemoteAdd(name, url string) *command.Model {
 
 // Fetch downloads objects and refs from another repository.
 func (g *Git) Fetch(opts ...string) *command.Model {
-	args := []string{"fetch"}
+	args := []string{"fetch", "--jobs=10"}
```

Comment:
Are we sure we want to hardcode 10 here?

Reply:
Based on the [action points](https://bitrise.atlassian.net/wiki/spaces/STEP/pages/1325236987/Git+Clone+Step+--jobs+flag+investigation#Suggested-follow-up) of the investigation, I believe yes.

Since it is only used for two commands, I don't see a point in extracting it to a const.

Reply:
I see but this is not the git clone step but a "generic" utility kind of lib. I think hardcoding 10 in git clone step might make sense however here I'd advise against it. wdyt?

Reply:
Makes sense! `fetch` already has an additional args, so we could just pass the `jobs` from the `git-clone` Step, and we can create a similar input for the `submodule update` as well.

Reply:
maybe introducing an overload FetchWithJobs() would be the safest choice. And the standard fetch just calls it with jobs = 0 and it adds the jobs argument only if it is more than 0.

That way we don't break the api

Reply:
Cool